paperqa/agents/env.py

```diff
@@ -151,21 +151,22 @@ def export_frame(self) -> Frame:
     async def step(
         self, action: ToolRequestMessage
     ) -> tuple[list[Message], float, bool, bool]:
+
+        # add usage for actions that have usage
```

Comment:
```suggestion
        # add usage if the action has usage
```

There's just one action